airflow/www/templates/airflow/tree.html

```diff
@@ -23,6 +23,10 @@
 <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/tree.css') }}">
 <link rel="stylesheet" type="text/css"
       href="{{ url_for('appbuilder.static',filename='datepicker/bootstrap-datepicker.css') }}">
+<meta name="data" content="{{ data }}"/>
```

Comment:
`| tojson`?

Reply:
I updated a PR. Now this PR use "| tojson | forceescape" to make the code more explict.